pkg/controller/usersignup/usersignup_controller.go

```diff
@@ -16,12 +22,15 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
-var log = logf.Log.WithName("controller_usersignup")
+const (
+	// Status condition reasons
+	noClustersAvailableReason = "NoClustersAvailable"
+	failedToReadUserApprovalPolicy = "FailedToReadUserApprovalPolicy"
+	failedToCreateMURReason = "FailedToCreateMUR"
```

Comment:
rather `FailedTo`, can we use `UnableTo` [link](https://github.com/codeready-toolchain/member-operator/blob/master/pkg/controller/useraccount/useraccount_controller.go#L34) for consistency. Both are good enough but the first wins ;-)

Reply:
Sure thing, I'll update this.